cilium/Makefile

```diff
@@ -7,6 +7,7 @@ SOURCES := $(shell find ../api ../daemon ../common ../pkg cmd . \( -name '*.go'
 $(TARGET): $(SOURCES)
 	@$(ECHO_GO)
 	$(QUIET)$(GO) build $(GOBUILD) -o $(TARGET)
+	strip $(TARGET)
```

Comment:
Doesn't this break panic backtraces? does it break anything at all?

Reply:
@raybejjani do we care about panics in the client?